pkg/sql/sem/tree/select.go

```diff
@@ -743,6 +744,18 @@ func (d Direction) String() string {
 	return directionName[d]
 }
 
+// ToIndexColumnDirection converts tree.Direction to catpb.IndexColumn_Direction.
+func (d Direction) ToIndexColumnDirection() catpb.IndexColumn_Direction {
+	switch d {
+	case DefaultDirection, Ascending:
+		return catpb.IndexColumn_ASC
+	case Descending:
+		return catpb.IndexColumn_DESC
+	default:
+		panic(fmt.Sprintf("invalid direction %s", d))
+	}
+}
+
```

Comment:
I don't like this dependency. Can we put this somewhere else? I would love if we could fully separate the catalog protobufs from the ast where neither know about each other. It'll take some time, but this is moving backwards. Currently tree doesn't import any of the protobufs.

Reply:
I'd expect the below rule to prevent this from passing. 

```
    disallowed_prefixes = [
        "pkg/sql/catalog",	        "pkg/sql/catalog",
    ],	    ],
```